sos/report/plugins/openstack_glance.py

```diff
@@ -70,7 +70,18 @@ def setup(self):
                                     "intended to connect to the OpenStack "
                                     "environment.")
             else:
-                self.add_cmd_output("openstack image list --long")
+                self.add_cmd_output(
+                    "openstack image list --long --all-projects"
+                )
+
+                cmd = "openstack image list -f value --all-projects"
+                glance_images = self.exec_cmd(cmd)['output']
```

Comment:
Same like in `cinder_volumes`. Again leaving on author's decision how safe it is..